msteams-platform/messaging-extensions/api-based-overview.md

```diff
@@ -382,68 +382,74 @@ Update app manifest (previously called Teams app manifest) with the `composeExte
 
 ```json
 {
-    "$schema": "https://developer.microsoft.com/json-schemas/teams/vDevPreview/MicrosoftTeams.schema.json",
-    "manifestVersion": "devPreview",
-    "version": "1.0.3",
-    "id": "55490e7f-xxxx-xxxx-xxxx-9eea9281b0fa",
-    "packageName": "com.microsoft.teams.extension",
-    "developer": {
-        "name": "Teams App, Inc.",
-        "websiteUrl": "https://www.example.com",
-        "privacyUrl": "https://www.example.com/termofuse",
-        "termsOfUseUrl": "https://www.example.com/privacy"
-    },
-    "icons": {
-        "color": "color.png",
-        "outline": "outline.png"
-    },
-    "name": {
-        "short": "sampletest2809-dev",
-        "full": "Full name for sampletest2809"
-    },
-    "description": {
-        "short": "Open AI Klarna product Api",
-        "full": "Full description for sampletest2809"
-    },
-    "accentColor": "#FFFFFF",
-    "composeExtensions": [
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.14/MicrosoftTeams.schema.json",
+  "manifestVersion": "devPreview",
+  "version": "1.0.4",
+  "id": "228a4704-xxxx-xxxx-xxxx-3b194021dd86",
+  "packageName": "com.microsoft.teams.extension",
+  "name": {
+    "short": "sampletest2809-dev",
+    "full": "Full name for sampletest2809"
+  },
+  "developer": {
+    "name": "Teams App, Inc.",
+    "websiteUrl": "https://www.example.com",
+    "privacyUrl": "https://www.example.com/termofuse",
+    "termsOfUseUrl": "https://www.example.com/privacy"
+  },
+  "description": {
+    "short": "Open AI Klarna product Api",
+    "full": "Full description for sampletest2809"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "accentColor": "#FFFFFF",
+  "composeExtensions": [
+    {
+      "composeExtensionType": "apiBased",
+      "apiSpecificationFile": "openapi.yml",
```

Comment:
This looks wrong:
.yaml not .yml

Also verify the file name as well.

Reply:
@ChetanSharma-msft  All the YAML files we use and also the format in the manifest is .yml and not YAML.

Sharing a screenshot of the manifest with .yml we tested and got the output.
<img width="278" alt="image" src="https://github.com/MicrosoftDocs/msteams-docs/assets/93579821/99d73fa4-a6d3-4abd-b52b-a976777aa03f">